### ISSUE TITLE
Add query logging to DozerDB

### ISSUE THREAD

**Issue by shaswatbedrock** [DozerDB/dozerdb-core#28]
Hi!

We were considering using DozerDB for our use case over Neo4j Enterprise edition as we prefer the open source nature of DozerDB. However, one feature we are really concerned about is the query logging. Is there any plan to add this to DozerDB?

Thanks!

**Comment by jmsuhy** [comment]
Yes, query logging is on our radar as we’ve had several end users ask about it.

I should have a timeline for when we can get it completed soon.

We will use this issue to track the progress.
